<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>test1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>test2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QtTreePropertyBrowser" name="propertyBrowser" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>qtpropertybrowser/qtvariantproperty.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
